docs/_toc.yml

```diff
@@ -8,8 +8,11 @@
 #   - file: introduction
 
 - part: Plotting
-  chapters: 
+  chapters:
   - file: plotting_3d
 - part : Summary Statistics
   chapters: 
-  - file: GraphStats
\ No newline at end of file
+  - file: GraphStats
+- part: Analyzing
```

Comment:
Maybe put Modularity as the title instead? I think each of our analyses will have their own section

Reply:
I just saw the discussion between Paul and Spencer in #39 . So I changed them under that Analysis section, if that sounds reasonable.

Reply:
I just saw that, you're right